src/SvelteTable.svelte

```diff
@@ -125,7 +127,10 @@
     filteredRows = asyncPagination ? [...rows] : rows.slice(from - 1, to);
   };
 
-  $: filteredRows = [...rows];
+  $: filteredRows = (() => {
+    lastSortedKey ? sortRowsBy(lastSortedKey, sortDescending) : undefined;
```

Comment:
also not a fan of `undefined` in a ternary, just a simple if statement would do either
```js
let filteredRows = [...rows];

$: {
	if(lastSortedKey) sortRowsBy(lastSortedKey, true);
	filteredRows = [...rows]
}
```
or
```js
$: filteredRows = (() => {
	if(lastSortedKey) sortRowsBy(lastSortedKey, true);
	return [...rows]
})();
```